src/sagemaker/jumpstart/utils.py

```diff
@@ -232,6 +232,19 @@ def add_single_jumpstart_tag(
     return curr_tags
 
 
+def get_jumpstart_base_name_if_jumpstart_model(
+    *uris: Optional[str],
+) -> Optional[str]:
+    """Return default JumpStart base name if a URI belongs to JumpStart.
+
+    If no URIs belong to JumpStart, return None.
```

Comment:
Can you please address Mufis' comment

Reply:
I did (`*uris (Optional[str]): URI to test for association with JumpStart.`). I believe you're looking at an old version.